app/src/androidTest/java/TestPlan.kt

```diff
@@ -21,72 +19,56 @@ class TestPlan : BaseTestPlan() {
-            NOTE_SCREEN_EDIT_TEXT_HINT_ELEMENT_THIRD,
-            getResourceString(R.string.format_hint_text)
-        )
+        addNoteScreen.run {
+            clickAddNoteButton()
+            verifySecondSpecificInputIsAndHasTextInput()
+        }
     }
 
     @Test
     fun checkNoteScreenAddItemEditText() {
-        clickBottomToolbarButton(HOME_PAGE_BOTTOM_BAR_ADD_NOTE_BUTTON_NUMBER)
-        clickBottomToolbarButton(NOTE_SCREEN_BOTTOM_BAR_ADD_ITEM_BUTTON_NUMBER)
-        checkHintOnEditInRecyclerView(
-            NOTE_SCREEN_EDIT_TEXT_POSITION_SECOND,
-            NOTE_SCREEN_EDIT_TEXT_HINT_ELEMENT_FOURTH,
-            getResourceString(R.string.format_hint_list)
-        )
+        addNoteScreen.run {
+            clickAddNoteButton()
+            clickThirdButtonOnBottomToolbar()
+            verifyThirdSpecificInputIsAndHasListInput()
+        }
     }
 
     @Test
     fun checkNoteScreenAddQuotedEditText() {
-        clickBottomToolbarButton(HOME_PAGE_BOTTOM_BAR_ADD_NOTE_BUTTON_NUMBER)
-        clickBottomToolbarButton(NOTE_SCREEN_BOTTOM_BAR_ADD_QUOTED_BUTTON_NUMBER)
-        checkHintOnEditInRecyclerView(
-            NOTE_SCREEN_EDIT_TEXT_POSITION_SECOND,
-            NOTE_SCREEN_EDIT_TEXT_HINT_ELEMENT_FOURTH,
-            getResourceString(R.string.format_hint_quote)
-        )
+        addNoteScreen.run {
+            clickAddNoteButton()
+            clickFourthButtonOnBottomToolbar()
+            verifyThirdSpecificInputIsAndHasQuoteInput()
+        }
     }
 
     @Test
     fun checkNoteScreenSmallHeadingEditText() {
-        clickBottomToolbarButton(HOME_PAGE_BOTTOM_BAR_ADD_NOTE_BUTTON_NUMBER)
-        clickBottomToolbarButton(NOTE_SCREEN_BOTTOM_BAR_EXTEND_MENU_BUTTON_NUMBER)
-        clickBottomToolbarButton(NOTE_SCREEN_BOTTOM_BAR_ADD_SMALL_HEADING_NUMBER)
-        checkHintOnEditInRecyclerView(
-            NOTE_SCREEN_EDIT_TEXT_POSITION_FIRST,
-            NOTE_SCREEN_EDIT_TEXT_HINT_ELEMENT_THIRD,
-            getResourceString(R.string.format_hint_heading)
-        )
+        addNoteScreen.run {
+            clickAddNoteButton()
+            clickFifthButtonOnBottomToolbar()
+            clickSecondButtonOnBottomToolbar()
+            verifyFirstSpecificInputIsAndHasHeadingInput()
+        }
     }
 
     @Test
     fun checkNoteScreenAddCodeEditText() {
-        clickBottomToolbarButton(HOME_PAGE_BOTTOM_BAR_ADD_NOTE_BUTTON_NUMBER)
-        clickBottomToolbarButton(NOTE_SCREEN_BOTTOM_BAR_EXTEND_MENU_BUTTON_NUMBER)
-
-        clickBottomHorizontalViewButton(
-            NOTE_SCREEN_BOTTOM_BAR_HORIZONTAL_NUMBER,
-            NOTE_SCREEN_BOTTOM_ADD_CODE_MENU_NUMBER
-        )
-        checkHintOnEditInRecyclerView(
-            NOTE_SCREEN_EDIT_TEXT_POSITION_SECOND,
-            NOTE_SCREEN_EDIT_TEXT_HINT_ELEMENT_THIRD,
-            getResourceString(R.string.format_hint_code)
-        )
+        addNoteScreen.run {
+            clickAddNoteButton()
+            clickFifthButtonOnBottomToolbar()
```

Comment:
Maybe better would be using function of these buttons instead of position?

Reply:
Up. I mean e.g. clickFavouriteNotesBottomBarButton instead clickFifthButtonOnBottomToolbar (favourite notes is just an example).